pools.json

```diff
@@ -535,6 +535,10 @@
         "/ArkPool/": {
             "name": "ArkPool",
             "link": "https://www.arkpool.com/"
+        },
+        "/PureBTC.COM/" : {
+            "name": "PureBTC.COM",
+            "link": "https://purebtc.com"
```

Comment:
0x4eC6bc9691F2742C86307dA42f1F0b280a7c5D0F

Reply:
?

Reply:
> ?

this is some kind of spam